src/GeneratedMapper/SyntaxReceivers/MapAttributeReceiver.cs

```diff
@@ -8,10 +9,15 @@ namespace GeneratedMapper.SyntaxReceivers
 {
     internal sealed class MapAttributeReceiver : ISyntaxReceiver
     {
-        public List<TypeDeclarationSyntax> Candidates { get; } = new List<TypeDeclarationSyntax>();
+        public List<TypeDeclarationSyntax> Candidates { get; } = new();
```

Comment:
I think we need to rename this to `AttributeCandidates`.

Reply:
Ok can do

Reply:
Didn't update this because it is originally Attribute candidates but I do append projection and extension candidates in there too with Trim, so it' all candidates.